controller/utils/api.py

```diff
@@ -81,6 +81,7 @@ def handler(**kwargs):
                 except ex.BadRequestException as e:
                     return bad_request(e)
                 except Exception as e:
+                    LOG.log(str(e))
```

Comment:
Adding log to every exception we have to facilitate debugging.

Reply:
That's great!